package.json

```diff
@@ -18,18 +18,20 @@
   "author": "DigiFi Team",
   "license": "ISC",
   "dependencies": {
+    "docusign-esign": "^8.0.1",
```

Comment:
i think we don't need to fetch full docusign-esign library just for types. And we probably should use our own interface in order to abstract docusign library

Reply:
agree, updated